google/cloud/storage/transfer_manager.py

```diff
@@ -329,6 +331,11 @@ def download_many(
         and the default is a conservative number that should work okay in most
         cases without consuming excessive resources.
 
+    :type skip_if_exists: bool
+    :param skip_if_exists:
+        Before downloading each blob, check if the file for the filename exists;
+        if it does, skip that blob. This only works for filenames.
```

Comment:
I think it's worth calling out that this is referring to `blob_file_pairs`  - can only be a list of tuples of blob and a filename for this to work. 

Will leave it up to you if you want to add a ValueError validation here. What I'm thinking is that a user may miss this note and not know why downloads are still repeated when passing in file-like objects

Reply:
Open file-like objects definitionally represent extant files, so hopefully no user should expect this feature to affect them. I'll pass on the validation for now; mixed filename/file handler lists are possible (honestly they cause only trouble so I should have banned them, but too late for that now) and it's complex to look ahead and only raise ValueError if all of them are identical.